README.md

```diff
@@ -31,4 +31,4 @@ You a given a list of items and markup for the `Pagination`. Implement the
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_pagination/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<Vasyl-Zhyliakov>.github.io/react_pagination/) and add it to the PR description.
```

Comment:
Your link in PR description doesn't work, use this.  [DEMO LINK](https://Vasyl-Zhyliakov.github.io/react_pagination/)
```suggestion
- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Vasyl-Zhyliakov.github.io/react_pagination/) and add it to the PR description.
```

![image](https://github.com/mate-academy/react_pagination/assets/70009205/24770d5b-705e-4e69-a0e1-543edc1e2aa0)